.github/workflows/build.yml

```diff
@@ -155,8 +162,9 @@ jobs:
           "LUA_SUBPROJECT_PATH=subprojects/lua-5.4.4" >> $env:GITHUB_ENV
       - name: Configure
         run: |
-          meson setup --wrap-mode=forcefallback build
+          meson subprojects download
           Get-Content -Path resources/windows/001-lua-unicode.diff -Raw | patch -d $env:LUA_SUBPROJECT_PATH -p1 --forward
+          meson setup --wrap-mode=forcefallback build
```

Comment:
Might be a good idea to add a comment here too explaining why `meson subprojects download` is needed before `setup`.

Reply:
Ok.